Makefile

```diff
@@ -1,20 +1,22 @@
 include ./make/*.mk
 
 ORG ?= integreatly
-NAMESPACE_PREFIX=redhat-rhmi-
+NAMESPACE_PREFIX ?= redhat-rhmi-
 NAMESPACE=$(NAMESPACE_PREFIX)operator
 PROJECT=integreatly-operator
+RHOAM_PROJECT=managed-api-service
```

Comment:
I guess this is the project in quay? can we use one PROJECT env and move it to the block below that manages the envs?

Reply:
Yes we can move it